src/cascadia/TerminalControl/TermControl.h

```diff
@@ -26,6 +26,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         TermControl(IControlSettings settings, TerminalConnection::ITerminalConnection connection);
 
         winrt::fire_and_forget UpdateSettings();
+        winrt::fire_and_forget UpdateAppearance(const IControlAppearance newAppearance);

```

Comment:
```suggestion
        winrt::fire_and_forget UpdateAppearance(const IControlAppearance& newAppearance);
```

always one more & ;)

Reply:
wait, we can't do this because its a `fire_and_forget` right?

Reply:
i keep making this mistake! AGH.